xml/System.Net.Sockets/Socket.xml

```diff
@@ -3802,7 +3802,7 @@
           <format type="text/markdown"><![CDATA[  
   
 ## Remarks  
- The `Connected` property gets the connection state of the <xref:System.Net.Sockets.Socket> as of the last I/O operation. When it returns `false`, the <xref:System.Net.Sockets.Socket> was either never connected, or is no longer connected.  
+ The `Connected` property gets the connection state of the <xref:System.Net.Sockets.Socket> as of the last I/O operation. When it returns `false`, the <xref:System.Net.Sockets.Socket> was either never connected, or is no longer connected. `Connected` is not thread-safe, it may return `true` after an operation is aborted due to an operation that disconnects the <xref:System.Net.Sockets.Socket> on another thread.
```

Comment:
should we mention this is transient e.g. short time race condition?

Reply:
I don't want to imply that people could poll this value. I'm not sure that is guaranteed to always work.

Reply:
ok, I assumed it would converge.